examples/annotation_import/tiled.ipynb

```diff
@@ -38,34 +38,52 @@
       },
```

Comment:
<p>Remove the unused imports please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done !